flink-runtime/src/test/java/org/apache/flink/runtime/operators/util/BloomFilterTest.java

```diff
@@ -168,4 +173,161 @@ public void testHashcodeInput() {
         assertTrue(bloomFilter.testHash(val4));
         assertTrue(bloomFilter.testHash(val5));
     }
+
+    @Test
+    public void testBloomFilterSerDe() {
```

Comment:
testBloomFilterSerializationAndDeserialization

Reply:
Can we migrate this test to junit5?

Reply:
Yes, I have migrated all tests to junit5